@xen-orchestra/lite/src/App/TabConsole.tsx

```diff
@@ -65,6 +66,20 @@ const TabConsole = withState<State, Props, Effects, Computed, ParentState, Paren
         </p>
       ) : (
         <>
+          <TitleBar actions={[
+            {
+              icon: 'pen',
+              color: "primary",
+              title: "edit",
+              variant:"contained"
+            },
+            {
+              icon: 'trash',
+              color: "warning",
+              title: "don't delete me, please",
+              variant:"outlined"
+            },
+          ]}>{state.vm?.name_label ?? 'loading'} </TitleBar>
```

Comment:
Yes, why not?
@pdonias What do you think?

Reply:
I don't think it's necessary to add it to the styleguide because it's a component that can't really be used on its own. But you can add a "start" action, use an internationalized label for it and trigger a "Not implemented" message when it's clicked.